dist/deno/types/merging.ts

```diff
@@ -182,3 +183,21 @@ export type DeepMergeBuiltInMetaData = Readonly<{
   key: PropertyKey;
   parents: ReadonlyArray<Readonly<Record<PropertyKey, unknown>>>;
 }>;
+
+export type DeepMergeLeafNoUndefinedOverride<
```

Comment:
Before fixing, I check https://github.com/RebeccaStevens/deepmerge-ts/discussions/25 for reference,, so I put this codes.

then I think its unnecessary to create another type declaration,
because the result type will be the same any way (Ts)

maybe I should remove this types declaration too?